baseframe/templates/baseframe/bootstrap3/forms.html.jinja2

```diff
@@ -156,9 +156,10 @@
     <script type="text/javascript">
       $(function() {
         // Disable submit button when clicked. Prevent double click.
-        $('#{{ ref_id }} input[type="submit"]').click(function() {
-          $(this).addClass('submit-disabled').prop('disabled', true);
-          $('#{{ ref_id }}').find(".loading").removeClass('hidden');
+        $('#{{ ref_id }}').submit(function() {
+          $(this).find('button[type="submit"]').addClass('submit-disabled').prop('disabled', true);
+          $(this).find('input[type="submit"]').prop('disabled', true).addClass('submit-disabled');
```

Comment:
Any reason the chaining order is reversed between these two lines?

Reply:
Earlier it was handling form submits for input types as submit. Now added the case to handle forms submit with buttons of type submit.